javaparser-symbol-solver-testing/A.java

```diff
@@ -0,0 +1 @@
+public class A { public int FUNC() { Class<?> cls = null; if(cls == null) return 0; else { B.FUNC3(cls); return 1;} } public void FUNC2(Class<?> arg) { return; }}class B {public static void FUNC3(Class<?> arg) { return; }}
```

Comment:
Enter at the end of the file! :-D

(and at the end of lines ;-) )

Reply:
ah ah, I deserve that. And should delete this file which was generated by an example provided by a user and should not be included :D